index.html

```diff
@@ -1,300 +1,288 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Portfolio</title>
+    <title>Personal Portoflio</title>
     <link rel="stylesheet" href="css/style.css" />
   </head>
   <body>
-    <header>
-      <a href="#" class="logo">
-        <img src="images/MyLogo.svg" alt="My Logo" />
-      </a>
-      <div class="union" id="union">
-        <img src="images/union.svg" alt="union icon" />
-      </div>
-    </header>
-    <section class="head-line">
-      <h1 class="introduction first-text1">
-        I<span>&#8217;</span>m Kumilachew<br />
-        Glad to see <br />
-        you!
-      </h1>
-      <p class="first-text2">
-        I'm a software developer I can help you build a product , feature or
-        website Look through some of my work and experience! If you like what
-        you see and have a project you need coded, don't hestiate to contact me.
-      </p>
-      <div>
-        <a class="link" href="#">
-          <h2 class="intro connect">LET'S CONNECT</h2>
-        </a>
-      </div>
-      <ul class="intro headd-icons">
-        <li>
-          <a href="#">
-            <img src="images/twitter.svg" alt="twitter" />
-          </a>
-        </li>
-        <li>
-          <a href="#">
-            <img src="images/linkedin.svg" alt="linkedin" />
-          </a>
-        </li>
-        <li>
-          <a href="#">
-            <img src="images/medium.svg" alt="medium" />
-          </a>
-        </li>
-        <li>
-          <a href="#">
-            <img src="images/github.svg" alt="github" />
-          </a>
-        </li>
-        <li>
-          <a href="#">
-            <img src="images/istock.svg" alt="istock" />
-          </a>
-        </li>
-      </ul>
-    </section>
-    <!-- Work Section -->
-    <section class="wrapper">
-      <div class="shot">
-        <div class="snapshot"><img src="images/Shot1.svg" alt="tonic" /></div>
-        <div class="block">
-          <h1 class="heading1">Tonic</h1>
-          <div class="subframe">
-            <p class="sub1">CANOPY</p>
-            <div class="circle"></div>
-            <p class="sub2">Back End Dev</p>
-            <div class="circle"></div>
-            <p class="sub2">2015</p>
-          </div>
-          <p class="normal">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
+    <div>
+      <!--The nav bar-->
+      <nav>
+        <h3 class="logo indicator">My Logo</h3>
+        <div>
+          <ul id="desk-list">
+            <li class="indicator">Portoflio</li>
+            <li class="indicator">About</li>
+            <li class="indicator">Contact</li>
+          </ul>
+          <img
+            class="indicator"
+            id="union-menu"
+            src="images/Union.svg"
+            alt="Union"
+          />
+        </div>
+      </nav>
+      <!--The main section-->
+      <main>
+        <section>
+          <h1>I'm Kumilachew Glad to see you!</h1>
```

Comment:
[Optional Requirement]
The colour of the `<h1>` text does not match the Figma design. Kindly fix it